Automattic/wp-calypso
# Remove wpcom plugins code from jetpack plugins

More cleanup for /plugins

This PR removes a component and some code that were being used to set up the good old wp.com business plugins, which were removed 9 months ago.

When they were removed, this component and the code triggering it stayed behind, even if there was no way they could get executed anymore.

So I'm removing it.

How to test it
==========
I'm not sure, since this code wasn't being executed anymore... I guess ... go to /plugins, select one and make sure that the plugins page, once you select a site, is still there and nothing is broken? :)
